1_fetch/src/fetch.R

```diff
@@ -0,0 +1,16 @@
+library(dplyr)
+library(readr)
+library(stringr)
+library(sbtools)
+
+pull_data <- function(outfile){
+  mendota_file <- file.path(outfile)
```

Comment:
file.path is just a string operation (doesn't change outfile if that's the only argument), so it's unnecessary now

Reply:
ohh. it's like `os.path.join` in python. Thanks.

Reply:
done